open-craft/sprints
# [BB-1603] Split Jira sprints between cells and create new sprints via dashboard

This handles cross-cell tickets on the dashboard and adds a functionality to create new sprints while ending the current ones. It currently uses a workaround for dealing with currently shared sprints. There will be a follow-up to remove temporary workarounds after ending the current sprint (please see the `TODO` comments).

## Testing instructions:
1. Set `.env` according to the ticket description.
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Create superuser with `docker-compose -f local.yml run --rm django python manage.py createsuperuser` (the button on frontend should be visible for `staff` users only).
1. Add email to your superuser account under `http://localhost:8000/admin/account/emailaddress/`.
1. Navigate to `frontend` and run `npm i && npm start`.
1. Log into the frontend.
1. Check that the dashboard is loading properly.
